lib/core/utils/dq-element.js

```diff
@@ -36,7 +37,10 @@ function getSource(element) {
  * @param {Object} options Propagated from axe.run/etc
  * @param {Object} spec Properties to use in place of the element when instantiated on Elements from other frames
  */
-function DqElement(elm, options = null, spec = {}) {
+const DqElement = memoize(function DqElement(elm, options, spec) {
```

Comment:
Default parameters get transformed out of the function after babel, which meant that only `elm` was left in the argument list for memoize, so `options` and `spec` were ignored.